docs/src/pages/components/data-grid/columns/ColumnFluidWidthGrid.tsx

```diff
@@ -28,6 +28,7 @@ export default function ColumnFluidWidthGrid() {
           },
         ]}
         rows={rows}
+        hideFooter
```

Comment:
I think that we should use as many of the default props as possible in the demos. If we aren't happy with them, then it could signal they are not the right default. Extra props also harm the clariy of the inline preview. The vertical space of the demo isn't very important.

```suggestion
```